dist/common/systemd/scylla-debugfs.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Scylla debugfs configrator
+After=local-fs.target
+
+[Service]
+Type=simple
+ExecStart=/usr/sbin/scylla_debugfs
+
+[Install]
+WantedBy=local-fs.target
```

Comment:
strictly speaking, i don't think local-fs should depend on this service. what should depend on it is, IMO, `scylla-server` or `multi-user`. what do you think?